language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/typecast_expr_ctx_config1.json

```diff
@@ -9,15 +9,15 @@
       "label": "ErrorName",
       "kind": "Event",
       "detail": "Error",
-      "sortText": "L",
+      "sortText": "AL",
```

Comment:
In this case `int value2 = <>`, are we planning to show `int` types sorted first? Or just `types` first?

Reply:
This is just focusing the types. We do not consider the fact whether we can cast or not due to the inconsistency of the API

Reply:
No issue then.